.nvmrc

```diff
@@ -0,0 +1 @@
+12.16.1
```

Comment:
@kyle-bowden I think if this one is for sbt-accessibility-linter we should use 14.18.0 https://github.com/hmrc/sbt-accessibility-linter/blob/main/.nvmrc#L1